resonant-circuit/capacitor-matlab/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Import MATLAB-bindings path
+. ../matlab-bindings-path.sh
+
+# Add bindings to MATLAB path
+export MATLABPATH="$BINDINGS"
+
+# Run MATLAB code without GUI
+LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6 matlab -nodisplay -nosplash -nodesktop -r "capacitor;exit;"
```

Comment:
```suggestion
LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6 matlab -nodisplay -r "capacitor;exit;"
```
Don't want to discuss `LD_PRELOAD` here, but I think we can remove some additional arguments. Only `-nodisplay` is needed.